sys/auto_init/auto_init.c

```diff
@@ -284,42 +284,106 @@ void auto_init(void)
 #ifdef MODULE_AUTO_INIT_SAUL
     DEBUG("auto_init SAUL\n");
 
-#ifdef MODULE_SAUL_GPIO
-    extern void auto_init_gpio(void);
-    auto_init_gpio();
-#endif
 #ifdef MODULE_SAUL_ADC
     extern void auto_init_adc(void);
     auto_init_adc();
 #endif
-#ifdef MODULE_LSM303DLHC
-    extern void auto_init_lsm303dlhc(void);
-    auto_init_lsm303dlhc();
+    #ifdef MODULE_SAUL_GPIO
```

Comment:
There's an extra indent here.

Reply:
Oops :worried:

Reply:
Done. May I reset the branch to commit sys/auto_init/auto_init.c separately?